code/modules/surgery/organs/internal/heart/_heart.dm

```diff
@@ -66,34 +97,32 @@
 	if(!owner.needs_heart())
 		return
 
-	if(owner.client && beating)
-		failed = FALSE
-		var/sound/slowbeat = sound('sound/health/slowbeat.ogg', repeat = TRUE)
-		var/sound/fastbeat = sound('sound/health/fastbeat.ogg', repeat = TRUE)
+	// Handle "sudden" heart attack
+	if((organ_flags & ORGAN_FAILING) || !beating)
```

Comment:
```suggestion
	if(!beating || (organ_flags & ORGAN_FAILING))
```

this is a really bad nit but i like it ending in `))` rather than leading with `((`... i might have a problem so feel free to ignore this one